programmers/크레인-인형뽑기-게임.js

```diff
@@ -0,0 +1,20 @@
+function solution(boards, moves) {
+    const basket =[];
+    const indexMoves = moves.map((x) =>  (x - 1));
+    let cnt = 0;
+
+    for (let move of indexMoves) {
+         for (let board of boards) {
+             if (board[move] !== 0) {
+               if (basket[basket.length - 1] === board[move]) {
+                     basket.pop();
+                     cnt += 2; 
+                  }
+               else basket.push(board[move]);
+               board[move] = 0;
+               break;
+             }
+         }
+     }
+    return cnt;
+}
```

Comment:
오마이갓... 들여쓰기가 .... 최악이야... 공백이랑 같이 많이많이  신경써야겠다 
```suggestion
function solution(boards, moves) {
  const basket = [];
  const indexMoves = moves.map((x) => x - 1);
  let cnt = 0;

  for (let move of indexMoves) {
    for (let board of boards) {
      if (board[move] !== 0) {
        if (basket[basket.length - 1] === board[move]) {
          basket.pop();
          cnt += 2;
        } else basket.push(board[move]);

        board[move] = 0;
        break;
      }
    }
  }
  return cnt;
}
```